es-design-system/pages/molecules/es-video.vue

```diff
@@ -1,18 +1,21 @@
-<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
 <template>
     <div>
         <h1>
             Video
         </h1>
         <p>
-            Shows the embed video when the cover is clicked.
+            Shows a YouTube video when the provided cover image is clicked.
         </p>
         <div>
-            <es-video
-                :alt-text="altText"
-                class="my-450 mb-200 mb-lg-0"
-                :cover-image-url="placeholderEsThumbnail"
-                :embed-url="embedUrl" />
+            <b-row class="justify-content-center">
+                <b-col md="8">
+                    <es-video
+                        alt-text="Community solar: Everything you want to know before you join"
+                        class="my-450 mb-200 mb-lg-0"
```

Comment:
Couple small things:
- The spacing looks great on desktop but is a bit large on mobile - maybe we can do `my-200 my-lg-450`?
- Because of the `my-450`, I don't think the `mb-200 mb-lg-0` is doing anything, since `my` covers both `mb` and `mt`, so we can probably remove those.
- To make the `<es-video>` example easier to copy out for any use case, might be worth moving the margin class(es) up to the `<b-row>` element.